### ISSUE TITLE
Annona - Accessibility - Buttons must have discernible text

### ISSUE THREAD

**Issue by uom-mde** [NCSU-Libraries/annona#53]
Hello all,

Firstly, many thanks for making this superb feature available- we've used it extensively in our latest online exhibition:
https://www.digitalexhibitions.manchester.ac.uk/s/qing/

While running accessibility checks on our site, they return a 'Buttons do not have an accessible name' flag. Is this something that can be addressed?

Best wishes,
John

**Comment by uom-mde** [comment]
Hi Niqui,

Many thanks for your swift response, and for offering to make these changes. Apologies for confusing matters and not linking directly to a page using Annona: we have used both Exhibit.so and Annona, with Exhibit featuring on the home page, and Annona on the majority of others.

For example, you can find Annona used on this page:
https://www.digitalexhibitions.manchester.ac.uk/s/qing/page/knowledge-exchange

Best wishes,
John

**Comment by dnoneill** [comment]
Hi John,
I updated the toolbar so there should be an aria label that tells the user what each button does. I ran the WAVE extension on Annona before releasing it. I am wondering what you use for accessibility because it seems more robust. Please let me know if the latest update doesn't fix the problem. Also please let me know if there is anything else I can do to help. I love how you are using Annona.

Niqui

**Comment by uom-mde** [comment]
Hi Niqui,

Thanks for getting back to me, and for implementing that fix in double-quick time! I downloaded the core files again, generated a new 'dist' folder, and the aria labels are now present when running Annona according to the accessibility checkers.

For reference I use Google's Lighthouse tool, which can be added to Chrome as an extension. (I think it might be available by default within their developer tools, but can't confirm for certain....)

Thanks again for your/your team's great work on this feature- it's really appreciated- and your kind words on how we're applying it on our site. Should I note anything else I'll be in touch!

Best wishes,
John

**Comment by dnoneill** [comment]
John,
Thank for getting back to me. I am going to go ahead and close this issue. Glad you reached out. I am hoping if it is okay with you to put your site on our in the wild section: https://ncsu-libraries.github.io/annona/examples/. Have a nice day.

Niqui
